src/Microsoft.AspNetCore.Razor.Design/Microsoft.AspNetCore.Razor.Design.csproj

```diff
@@ -1,14 +1,12 @@
-﻿<Project>
-
-  <!-- Using explicit SDK imports here because the default way conflicts with the AfterBuild target -->
-  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
-
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>Razor is a markup syntax for adding server-side logic to web pages. This package contains MSBuild support for Razor.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
```

Comment:
Why this change? I don't fully understand.

Reply:
We want to able to target the two projects this project references. The TFMs here ultimately do not matter since there's no shipping code in this project.

Reply:
Got it. Please add a comment.